lib/srv/db/watcher.go

```diff
@@ -171,8 +171,19 @@ func (s *Server) matcher(resource types.ResourceWithLabels) bool {
 	if !ok {
 		return false
 	}
-	if database.IsRDS() || database.IsRedshift() {
+
+	// In the case of CloudOrigin RDS or Redshift DB resources the matchers should be skipped.
+	if hasOriginCloudLabel(resource) && (database.IsRDS() || database.IsRedshift()) {
```

Comment:
I think we can completely drop `IsRDS` and `IsRedshift`, or maybe replace it with `IsCloudHosted` if want to play safe.

Reply:
nit: This is probably not very useful, but I was curries if this feature works only in markdown files or also in comments.
```mermaid
graph
    A((IsCloudHosted)) --> B((IsAWS))
    A((IsCloudHosted)) --> C((IsGCP))
    A((IsCloudHosted)) --> D((IsAzure))
    B --> E((IsRDS))
    B --> F((IsRedshift))
    B --> G((IsElasticache))
    C --> H((IsCloudSQL))
```